modules/postgres/main.tf

```diff
@@ -1,11 +1,29 @@
+# Create a security group allowing access to the database port.
+resource "aws_security_group" "postgresql" {
+  vpc_id = "${var.vpc_id}"
+  name   = "${var.SecurityGroupName}"
```

Comment:
from, at least, the documentation tf vars tend to be snake_case so that's the style I've adopted for my vars. Do you feel like we should be consistent from the start? This is not a blocker for the PR for me, just a question

Reply:
I totally agree that they should be that in the end.  It's something I meant to mention -- basically right now I'm just taking the variable names as they are in the existing cloudformation setup to bring over, for clarity when I compare the configurations.  That's why vpc_id is the terraformish variable, because it didn't come from the old setup.

I do want to convert over -- using this style of variables makes my eyes bleed in terraform where yup, they definitely do not belong.  I mostly just wasn't sure of when I wanted to convert over and was leaving that as more of a general thing for once we had most of the build under our belt in case we needed to do more comparisons to the older code as we went on.  But I have no real attachment and could do this earlier.